config/locales/en/providers.yml

```diff
@@ -424,6 +425,16 @@ en:
           options:
             different_home_address: They have a different home address
             no_fixed_residence: My client is homeless or has no fixed residence
+      non_uk_home_addresses:
+        show:
+          heading: Enter your client's overseas home address
+          country: Country
+          address: Address
+          address_line_one: Address line 1
+          address_line_two: Address line 2 (optional)
+          address_line_three: Address line 3 (optional)
+          address_line_four: Address line 4 (optional)
+          submit_button: Save and continue
```

Comment:
Our pattern has been to use the `save_and_continue` key and value from the generic yml file